web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -188,6 +188,10 @@
   "Last refresh: {{time}}": "Last refresh: {{time}}",
   "See less": "See less",
   "See more": "See more",
+  "Filtered sum of top-k bytes / filtered total bytes": "Filtered sum of top-k bytes / filtered total bytes",
+  "Filtered top-k byte rate / filtered total byte rate": "Filtered top-k byte rate / filtered total byte rate",
+  "Filtered byte rate": "Filtered byte rate",
+  "Filtered sum of top-k packets / filtered total packets": "Filtered sum of top-k packets / filtered total packets",
```

Comment:
Should we keep "Filtered" mention when we don't have filters ?

Reply:
We could have something more detailed per context, e.g.:
- For flow queries: "Filtered sum of bytes" => "Sum of bytes from resulting flows"
- For topology queries: "Filtered sum of bytes" => "Sum of bytes from resulting graph"
- For overview queries: "Filtered sum of top-k bytes / filtered total bytes" => "Sum of bytes from resulting top-k metrics / from resulting total"

I don't know, it doesn't sound perfect either, I'm glad if someone comes up with a better proposal